poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -53,33 +61,35 @@ impl Heartbeats {
             hotspot_key: PublicKey,
             cbsd_id: String,
             reward_weight: Decimal,
-            timestamps: Vec<NaiveDateTime>,
+            hours_seen: [bool; 24],
         }
 
-        let heartbeats = sqlx::query_as::<_, HeartbeatRow>("SELECT * FROM heartbeats")
-            .fetch_all(exec)
-            .await?
-            .into_iter()
-            .map(|hb| {
-                (
-                    HeartbeatKey {
-                        hotspot_key: hb.hotspot_key,
-                        cbsd_id: hb.cbsd_id,
-                    },
-                    HeartbeatValue {
-                        reward_weight: hb.reward_weight,
-                        timestamps: hb.timestamps,
-                    },
-                )
-            })
-            .collect();
+        let heartbeats = sqlx::query_as::<_, HeartbeatRow>(
+            "SELECT (hotspot_key, cbsd_id, reward_weight, hours_seen) FROM heartbeats",
```

Comment:
this is the same as `*` right? this diff looks like it's changing more than i think it really is; effectively you're only actually accessing the `hours_seen` field on the result instead of the `timestamps` field previously?